src/Maestro/Maestro.Data/DarcRemoteFactory.cs

```diff
@@ -142,8 +141,11 @@ public async Task<string> EnsureLocalGit(ILogger logger)
                             }
 
                             logger.LogInformation($"Extracting '{gitZipFile}' to '{targetPath}'");
-
+#if NETCOREAPP2_1
                             ZipFile.ExtractToDirectory(gitZipFile, targetPath, overwriteFiles: true);
+#else
+                            ZipFile.ExtractToDirectory(gitZipFile, targetPath);
```

Comment:
Where is the not core version of this assembly getting used?  The behavior of the 'else' doesn't match the if, because overwriteFiles: true isn't the default. So this this is going to weirdly throw in the not-core scenarios, which seems bad.  Because I'm pretty sure "standard" isn't "NETCOREAPP2_1", so the you might have a core version of the app, but still get broken behavior because of this if during compilation.

Reply:
This popped up because the logic was written in SubscriptionActorService, which only targets core & the override doesn't exist in other versions. How would you recommend handling the move to a shared project? (Or is the best bet to pull out the method & leave it in the original project? As far as I know it's not needed for the work I'm doing, given that it was added afterwards.)

Reply:
I don't think we... need... to be "standard" anywhere.  There are only a few projects that are, for the time being, "net", which is the only time standard is useful.  We should probably just move away from "netstandard" and just use net core everywhere, for everything, except the few libraries that need to be used by the cloud services of the Helix API (since those are the only non-core things we have).

This is _very_ important that override is true. Otherwise this is going to crash in a singleton instance and possibly never recover (if there is some reason it can't delete one of the temp files)... crashing the process over and over.